posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,276 @@
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot-off-the-press link:https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]. Jakarta EE 10 is the first release to update the Jakarta EE specifications since Java EE 8 in 2017. While the Jakarta EE 10 release is still around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  The features that make up the new Jakarta EE 10 Core Profile are available in this beta release. 
+
+* Jakarta RESTful Web Services 3.1: support multipart/form-data.
+* Jakarta JSON-B 3.0: polymorphic serialization/deserialization; deserialization of null to JsonValue.NULL, etc.
+* Jakarta JSON-P 2.1: handling duplicated keys; more interfaces for primitives, lists and objects 
+* Jakarta CDI 4.0: defining new extension APIs; removing reflections; adding new reflection-free language model; empty beans.xml means that only annotated classes are beans; removal of deprecated APIs
+** Jakarta Annotation 2.1: allowing `@Priority` to be used on any program element; adding `@Nullable` and `@NotNull`
+** Jakarta Interceptors 2.1: picking up the changes from Annotation 2.1
+
+In addition to the Core Profile features, Jakarta EE 10 Expression Language 5.0 is included in this beta and provides new features and specification clarifications.   
+
+But it's not only Jakarta EE features that are debuting in this release!  MicroProfile OpenAPI is updated to version 3.1, in which you can specify optional authentication for operations and multiple annotation enhancements.  Last but not least, Password Utilities 1.1 is introduced in this beta, which no longer forcefully federates stand-alone user registries.
+
+
+The link:{url-about}[Open Liberty] 22.0.0.9-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* Jakarta EE 10 Core Profile
+** <<restful, RESTful Web Services 3.1>>
+** <<json, Jakarta JSON Binding and Processing>>
+** <<cdi, CDI 4.0>>
+* <<expression, Jakarta EE 10 Expression Language 5.0>>
+* <<microprofile, MicroProfile OpenAPI 3.1>>
+* <<password, Password Utilities 1.1>>
+
+[#restful]
+== RESTful Web Services 3.1
+
+RESTful Web Services 3.1 (also known as Jakarta Rest 3.1 and formerly known as JAX-RS) is now available in the Open Liberty 22.0.0.9 beta. The 3.1 implementation is based on RESTEasy and is  an EE10 technology, which uses the `jakarta.*` package name. For more information on the changes in this release, see the link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html#changes-since-3.0-release[Changes Since 3.0 Release section] of the link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html[Jakarta RESTful Web Services Spec].
```

Comment:
```suggestion
Jakarta RESTful Web Services 3.1 (also known as Jakarta Rest 3.1 and formerly known as JAX-RS) is now available in the Open Liberty 22.0.0.9 beta. The 3.1 implementation is based on RESTEasy and is  an EE10 technology, which uses the `jakarta.*` package name. For more information on the changes in this release, see the link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html#changes-since-3.0-release[Changes Since 3.0 Release section] of the link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html[Jakarta RESTful Web Services Spec].
```